docs/developer_guide.md

```diff
@@ -53,6 +53,9 @@ A full list of all areas can be found in the **FeatureArea.props** files. To inc
 <FeatureRepeaterEnabled>productOnly</FeatureRepeaterEnabled>
 ```
 
+When switching between the **MUXControls.sln** and **MUXControlsInnerLoop.sln**, make sure you either delete the **BuildOutput** folder in the solution's root directory or run `git clean -dfx`.
+Note that `git clean -dfx` deletes all untracked files, so it may also delete any changes you made. To see what would be deleted by that command without actually deleting anything, run `git clean -dfxn`.
+
```

Comment:
Maybe we should also point out why this is needed (i.e. that it get's rid of files that MSBUILD clean does not remove)?

Reply:
@yoshiask I think this is a good suggestion :)